sap-spring-properties/src/main/java/com/sap/cloud/environment/servicebinding/environment/SapServiceBindingEnvironmentPostProcessor.java

```diff
@@ -0,0 +1,37 @@
+package com.sap.cloud.environment.servicebinding.environment;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.context.properties.bind.BindResult;
+import org.springframework.boot.context.properties.bind.Bindable;
+import org.springframework.boot.context.properties.bind.Binder;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.annotation.Order;
+import org.springframework.core.env.ConfigurableEnvironment;
+
+import com.sap.cloud.environment.servicebinding.SapServiceBindingsProperties;
+
+/* An {@link EnvironmentPostProcessor} that binds service binding properties to {@link SapServiceBindingsProperties}. */
+@Order
+public class SapServiceBindingEnvironmentPostProcessor implements EnvironmentPostProcessor
+{
+    private static final Logger log = LoggerFactory.getLogger(SapServiceBindingEnvironmentPostProcessor.class);
+
+    @Override
+    public void postProcessEnvironment( ConfigurableEnvironment environment, SpringApplication application )
+    {
+        Bindable<SapServiceBindingsProperties> bindable = Bindable.of(SapServiceBindingsProperties.class);
+
+        BindResult<?> bindResult = Binder.get(environment).bind("cds", bindable);
```

Comment:
Please don't use the `cds` subsection, as this library is independent of CAP Java.

Reply:
well, I could not think of any usage of this library outside CAP and this provides a clearer structure of the application properties.

anyway, what about "services" (like the environment variable "VCAP_SERVICES")

Reply:
`services` or `bindings` sounds good